osquery/tables/events/tests/linux/seccomp_events_tests.cpp

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#ifdef __linux__
```

Comment:
You shouldn't need this `ifdef` but you will need to add this file as a new test in `./osquery/events/tests/CMakeLists.txt`.